### PR TITLE
Update cs.json

### PR DESCRIPTION
**Pull request by markrieder**
**Changes:**

**Testing:**

### Downstream

|Affects         |Reasoning
|----------------|-------------------------------
|Documentation   |

### CODE REVIEW

File: src/assets/i18n/cs.json [hunk 1]
```diff
@@ -329,7 +329,127 @@
   "Allow group members to use <a href=\"https://man7.org/linux/man-pages/man8/sudo.8.html\" target=\"_blank\">sudo</a>. Group members are prompted for their password when using <b>sudo</b>.": "",
   "Allow more ciphers for  <a href=\"https://man7.org/linux/man-pages/man8/sshd.8.html\" target=\"_blank\">sshd(8)</a>  in addition to the defaults in  <a href=\"https://man7.org/linux/man-pages/man5/sshd_config.5.html\" target=\"_blank\">sshd_config(5)</a>.  <code>None</code> allows unencrypted SSH connections and  <code>AES128-CBC</code> allows the 128-bit  <a href=\"https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.197.pdf\" target=\"_blank\">Advanced Encryption Standard</a>.<br><br>  WARNING: these ciphers are considered security vulnerabilities and  should only be allowed in a secure network environment.": "",
   "Allow non-root mount": "",
-  "Allow non-unique serialed disks (not recommended)": "",
+  "Allow non-unique serialed disks (not recommended)": "  "Admin Password": "Heslo administrátora",
```
Comment (undsoft) [inline src/assets/i18n/cs.json:332]: Syntax error here.

File: src/assets/i18n/cs.json [hunk 2]
```diff
@@ -329,7 +329,127 @@
...
+  "Allow Local User Login": "Povolit přihlášení lokálního uživatele",
+  "Allow Password Authentication": "Povolit ověřování heslem",
+  "Allow Specific": "Povolit specifické",
+  "Allow TCP Port Forwarding": "Povolit přesměrování TCP portů",
+  "Allow Taking Empty Snapshots": "Povolit pořizování prázdných snímků",
+  "Allow Transfer Resumption": "Povolit pokračování přenosu",
+  "Allow Trusted Domains": "Povolit důvěryhodné domény",
+  "Allow all initiators": "Povolit všechny iniciátory",
+  "Allow all sudo commands": "Povolit všechny sudo příkazy",
+  "Allow all sudo commands with no password": "Povolit všechny sudo příkazy bez hesla  "Admin Password": "Heslo administrátora",
```
Comment (undsoft) [inline src/assets/i18n/cs.json:392]: Syntax error.

**Review by undsoft — CHANGES_REQUESTED** [review]
Thank you for your contribution. There are a couple of syntax mistakes in your json file.

**Review by AlexKarpov98 — COMMENTED** [review]
Agree with Evgeny.

**Comment by AlexKarpov98** [comment]
@undsoft  @markrieder - that's the maximum I was able to fix.

This PR https://github.com/truenas/webui/pull/10316/files has more translations, but they all are incorrectly formatted, and I don't know how to inject them properly, and seems it'll be easier to add them again, instead of trying to understand where to inject which phrase.

Screenshot 2024-09-13 at 12 16 38

It's better to create a new PR, for the missing translations, please. 

@markrieder - thank you for your contribution. 
Please go ahead and add more translations, now you have a correct understanding how it works ✊

**Review by undsoft — APPROVED** [review]

**Pull request merged**